Patches/WWII Soviet Faction/WWIISovietFaction_CE_Patch_Headgear.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>WWII Soviet Faction (1.0)</modName>
+			</li>
+
+			<!-- ========== M73Cap =========== -->	
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M73Cap"]/statBases</xpath>
+				<value>
+					<Bulk>1</Bulk>
+					<WornBulk>0.75</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73Cap"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
```

Comment:
See M73 uniform comment above; should probably be between 0.01 - 0.08

Reply:
Reference photos suggest similar fabric thickness, so I'll use 0.05 as well.

Reply:
Fixed in e9d35bc